templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListForm/list-form.component.ts

```diff
@@ -5,20 +5,15 @@
   templateUrl: './list-form.component.html'
 })
 export class ListFormComponent implements OnInit {
-  _textField: string;
+  textField: string;
   @Output() inputTextEventEmitter = new EventEmitter<string>();
-  get textField() {
-      return this._textField;
-  }
-  set textField(textInput) {
-      this._textField = textInput;
-  }
+
   constructor() { }
```

Comment:
How come this closing bracket isn't on a new line like the other ones? Or is constructor a special case

Reply:
I don't think there is something special with constructor. But, this is normally the pattern I see for methods with no content.